### DISCUSSION TITLE
Add Ukrainian to your site

### DISCUSSION THREAD

**Discussion by seesmof** [dailydotdev/daily#1030] [Feature Requests]
Hey guys, I would like to ask yall to add Ukrainian translation, or let me translate your site into Ukrainian for absolutely free. Just give me the tools to do it and I will. I have experience in translating other apps on Crowdin so we can use that one. I'm sure this will help me and all the other Ukrainians that use your app. Thanks.

**Comment by idoshamun** [comment]
Hi @seesmof,

I appreciate your suggestion and willing to contribute the translation.
Unfortunately, our system doesn't support localization at the moment, so we can't provide you with any tools to do it 😢

**Reply by seesmof** [reply]
Hey @idoshamun,

Understood. Well, if it ever does, please reach out to me and I will happily provide a translation.

🇺🇦 🤙
